goagen/codegen/workspace.go

```diff
@@ -310,7 +310,7 @@ func PackagePath(path string) (string, error) {
 		if gp, err := filepath.Abs(gopath); err == nil {
 			gopath = gp
 		}
-		if strings.HasPrefix(absPath, gopath) {
+		if filepath.HasPrefix(absPath, gopath) {
```

Comment:
This is what I've done in other places for this:
https://github.com/egonelbre/goa/blob/44b6500703786f02032da6c00b5725aa1a9cb0e1/goagen/codegen/workspace.go#L102-L103
Maybe there's a better way...